Makefile

```diff
@@ -1,7 +1,13 @@
 NOCURL := 0
-STATIC := 1
+STATIC := 1  
+UNAME := $(shell uname)
 
-CURL   := E:\Dev\eclipse-workspace\curl
+ifeq ($(UNAME),Darwin) #Mac OS
+  NOCURL := 1
+  STATIC := 0
+endif
+
+CURL   := /usr/bin/curl

```

Comment:
If you are using "nocurl" for osx, why then change the CURL macro?
Also, CURL macro is location of the curl libraries /usr/bin/curl is the executable so this breaks building with curl